src/quadbez.rs

```diff
@@ -92,6 +92,55 @@ impl QuadBez {
     }
 }
 
+/// An iterator for quadratic beziers.
+pub struct QuadBezIter {
+    quad: QuadBez,
+    ix: usize,
+}
+
+impl Shape for QuadBez {
+    type PathElementsIter = QuadBezIter;
+
+    #[inline]
+    fn path_elements(&self, _tolerance: f64) -> QuadBezIter {
+        QuadBezIter { quad: *self, ix: 0 }
+    }
+
+    /// The area under the curve.
+    ///
+    /// We could just return 0, but this seems more useful.
```

Comment:
I think I'd prefer 0, to be consistent with Line. The idea of these individual signed_area methods is that you can add them up for all the segments of a closed path.

I'm wondering now whether it might be better do implement it for all such segments, that way you get the "shoelace algorithm" for polygon area. In any case, I'd prefer that Line, QuadBez, and CubicBez are consistent. I'm fine with PathSeg having the same behavior as a single element BezPath though.